deploy/olm-catalog/qdrouterd-operator/0.1.0/catalog-source.yaml

```diff
@@ -12,8 +12,7 @@ items:
           kind: ClusterServiceVersion
           metadata:
             annotations:
-              alm-examples: >-
-                [{"apiVersion":"interconnectedcloud.github.io/v1alpha1","kind":"Qdrouterd","metadata":{"name":"amq-interconnect"},"spec":{"count":2,"deploymentMode": "lbfrontend","image":"quay.io/interconnectedcloud/qdrouterd:1.5.0"}}]
+              alm-examples: '[{"apiVersion":"interconnectedcloud.github.io/v1alpha1","kind":"Qdrouterd","metadata":{"name":"amq-interconnect"},"spec":{"addresses":[{"distribution":"balanced","prefix":"balanced"},{"distribution":"closest","prefix":"closest"},{"distribution":"multicast","prefix":"multicast"}],"count":3,"deploymentMode":"lbfrontend","image":"quay.io/interconnectedcloud/qdrouterd:1.5.0"}}]'
```

Comment:
hmm, why the change in format?

Reply:
changed to use the original one. (this new one was created with csv gen. for taking reference purpose)